src/ActionList/List.tsx

```diff
@@ -242,4 +248,6 @@ export function List(props: ListProps): JSX.Element {
       })}
     </StyledList>
   )
-}
+})
+
+List.displayName = 'ActionList'
```

Comment:
Related: Should we set `Item`, `Group`, and `Header` `displayName` to `ActionList.Item`, `ActionList.Group`, and `ActionList.Header`, respectively?

Reply:
I think that's a great idea.  I _had_ to do it here because of the `forwardRef`, but I think adding those others would be a great separate PR